### ISSUE TITLE
Accessibility Testing with Fable

### ISSUE THREAD

**Issue by Abi-Nada** [cds-snc/platform-forms-client#2300]
### Description

We need to coordinate accessibility testing with Fable, for the unauthenticated form building experience. The objective is to identify what type of data is useful for this study (i.e. feedback regarding functional issues using assistive technology) and what data is not useful (i.e. language that doesn’t resonate with the tester - they’re a different user group than our clients).

### Acceptance Criteria:

- [ ] Identify what type of data is useful for this study
- [ ] Review prioritization with product

**Comment by connorscarolyns** [comment]
A single unmoderated test has been conducted on form building. @srtalbot we should have a small watch party to determine what is a useful insight to collect and how the test can be adjusted for future tests.

**Comment by connorscarolyns** [comment]
The watch party is completed, and we will move ahead with five more tests. They will focus on our most frequently used technology combinations: PC, Edge, Chrome, JAWS, IVDA, Magnification & Keyboard ATs.

**Comment by connorscarolyns** [comment]
Tests have been completed, videos have been tagged, and a watch party will happen at a future date (TBD). Closing this ticket for now,  will add a new one for a watch party
